File: src/main/java/Level1/Ex20.java
package Level1;

//정수 제곱근 판별
public class Ex20 {
    public static long solution(long n) {
        long answer = 0;
        for (int i = 0; i < Math.sqrt(n) + 1; i++) {
            if (Math.sqrt(n) == i) {
                answer = (long) (i + 1) * (i + 1);
            } else {
                answer = -1;
            }
        }
        return answer;
    }

    public static void main(String[] args) {
        long n = 3;
        System.out.println(solution(n));
    }
}

File: src/main/Javascript/Level1/Ex20.js
function solution(n) {
    var answer = 0;
    for (let i = 0; i < Math.sqrt(n) + 1; i++) {
        if (Math.sqrt(n) === i) {
            answer = Math.pow(i + 1, 2);
        } else {
            answer = -1
        }
    }
    return answer;
}

console.log(solution(121));
console.log(solution(3));